<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>usermodifypassword</class>
 <widget class="QDialog" name="usermodifypassword">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog
{
	background-image: url(:/images/mainBackgound.png);
}
QWidget
{
  	font-size:18px;
}
QPushButton
{
	font-size:18px;
	font-family:Microsoft Yahei;
}
QPushButton
{
background-color: rgb(192, 192, 224);
border-color: rgb(176, 186, 221);
border-width: 2px;
border-style: solid;
border-radius: 4;
}
QPushButton:hover
{
border-color: rgb(240, 240, 240);
}
QPushButton:pressed
{
		border-color: rgb(240, 240, 240);
}
QLineEdit
{
border-width: 0px;
border-style: solid;
border-radius: 4;
background-color:rgb(89, 107, 169);
selection-background-color:rgb(136,150,199);
color:white;
}
QCheckBox
{
  	font-size:18px;
	color: rgb(255, 255, 255);
	font-family:Microsoft Yahei;
}
QLabel
{
font-size:18px;
color: rgb(255, 255, 255);
font-family:Microsoft Yahei;
}
QGroupBox
{
font-size:20px;
font-family:Microsoft Yahei;
}
QTreeView::item{height: 35px; }

QTableWidget,QTableView,QTreeView{
background: rgb(229,227,231);
selection-color:black;
}
QHeaderView{
	background-color: rgb(201, 212, 242);
	border-bottom:1px;
	border-style:solid;
	border-color: rgb(160, 160, 160);
}
QHeaderView::section{
	height:25px;
	margin-top:5px;
	margin-bottom:5px;
	background-color: rgb(201, 212, 242);
	color:black;
	border:0px;
	border-right:1px solid;
	border-color:rgb(94, 98, 143);
	}
QTableWidget::item{
	height:45px;
	border-bottom:1px solid;
	border-color: rgb(198, 207, 214);
}
QTableWidget::item:selected{
	background-color: rgb(175, 211, 237);
}</string>
  </property>
  <widget class="QPushButton" name="pushButtonOK">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>400</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>400</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditOldPasswords">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>421</width>
     <height>41</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>旧密码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_newPasswords">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>151</y>
     <width>421</width>
     <height>41</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>151</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>新密码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_newPassword2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>230</y>
     <width>421</width>
     <height>41</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>42</x>
     <y>230</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>确认密码：</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
